packages/dds/tree/src/change-family/changeEncoder.ts

```diff
@@ -4,9 +4,11 @@
  */
 
 import { bufferToString, IsoBuffer } from "@fluidframework/common-utils";
+import { Jsonable } from "@fluidframework/datastore-definitions";
 
 export abstract class ChangeEncoder<TChange> {
     public abstract encodeForJson(formatVersion: number, change: TChange): JsonCompatible;
```

Comment:
I'm curious about naming asymmetry between 'encode*For*Json' and 'decodeJson'.  Was that intentional?

Reply:
@CraigMacomber actually wrote this file and picked these names, but the names seem fairly clear to me. I wouldn't object to something like `encodeAsJsonable` and `decodeFromJsonable`.